src/test/Productlist.test.js

```diff
@@ -0,0 +1,18 @@
+import { BrowserRouter } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import renderer from 'react-test-renderer';
+import store from '../REDUX/store';
+import ProductList from '../COMPONENTS/PRODUCTSLIST/Productlist';
+
+describe('to test the ProductList components', () => {
+  test('matches the ProductList DOM snapshot', () => {
+    const tree = renderer.create(
+      <BrowserRouter>
+        <Provider store={store}>
+          <ProductList />
+        </Provider>
+      </BrowserRouter>,
+    );
+    expect(tree).toMatchSnapshot();
+  });
```

Comment:
- [ ] [Optional] You have written the testing properly but when I run the test I got an error. See the image below for reference and please fix it
### Test error
![his](https://user-images.githubusercontent.com/81727988/189400428-4777870a-dc65-46bb-ab11-6063a6a53299.png)